projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.stats.blocking_reflect": "反伤",
+    "tetra.stats.blocking_reflect.tooltip": "§e§7防御近战伤害时有 §e%s§r 的概率对攻击者造成 §e%s§r 的伤害，并产生击退效果。",
+    "tetra.stats.bashing": "猛击",
+    "tetra.stats.bashing.tooltip": "按右键可以猛击生物。猛击会击退生物，击晕它并造成 §e100%%§r 的伤害\n \n§7击退倍率：§e%s§r\n§7击晕持续时间：§e%s§7 秒",
+    "tetra.stats.throwable": "可投掷",
+    "tetra.stats.throwable.tooltip": "可以按住右键来投掷。击中实体时造成 §e%s%%§r 的伤害。击中方块时，如果物品的采集等级足够高并且挖掘速度大于方块硬度，击中方块时会破坏此方块。",
+    "tetra.stats.jab": "猛刺",
+    "tetra.stats.jab.tooltip": "副手持有时可以右键攻击生物，造成 §e%s§r 点伤害并产生击中效果。触发等于物品攻击力的冷却时间。",
+    "tetra.stats.armorPenetration": "穿甲",
+    "tetra.stats.armorPenetration.tooltip": "击中目标将造成至少 §e%s§r 点伤害，无视护甲",
+    "tetra.stats.unarmoredDamage": "切割",
+    "tetra.stats.unarmoredDamage.tooltip": "攻击无护甲的生物时造成 §e%s§r 的额外伤害",
+    "tetra.stats.knockback": "击退",
+    "tetra.stats.knockback.tooltip": "增加 §e%s§r 倍的击退距离",
+    "tetra.stats.looting": "抢夺",
+    "tetra.stats.looting.tooltip": "增加杀死生物的掉落物",
+    "tetra.stats.fiery": "炽热",
+    "tetra.stats.fiery.tooltip": "攻击生物时使它着火 §e%s§r 秒",
+    "tetra.stats.smite": "亡灵杀手",
+    "tetra.stats.smite.tooltip": "在攻击亡灵生物时造成 §e%s§r 的额外伤害",
+    "tetra.stats.arthropod": "节肢杀手",
+    "tetra.stats.arthropod.tooltip": "在攻击节肢生物时造成 §e%s§r 的额外伤害并附加 1-§e%s§r 秒的 缓慢 IV 效果",
+    "tetra.stats.sweeping": "横扫之刃",
+    "tetra.stats.sweeping.tooltip": "攻击生物时也对所有附近的敌人造成 §e%s§r 的武器伤害",
+    "tetra.stats.unbreaking": "耐久",
+    "tetra.stats.unbreaking.tooltip": "使用时有几率不减少耐久\n \n§7耐久等级：§e%s§7\n耐久不减少概率：§e%s%%",
+    "tetra.stats.mending": "经验修补",
+    "tetra.stats.mending.tooltip": "消耗获得的经验球以修复耐久，每点经验修复 §e%s§r 耐久\n \n§7只在左手或右手持有时有效",
+    "tetra.stats.silkTouch": "精准采集",
+    "tetra.stats.silkTouch.tooltip": "破坏时掉落方块本身，而不是一般的掉落物",
+    "tetra.stats.fortune": "时运",
+    "tetra.stats.fortune.tooltip": "在采集或回收方块时增加掉落几率",
+    "tetra.stats.flame": "火矢",
+    "tetra.stats.flame.tooltip": "击中实体时使它着火 §e%s§r 秒，弹射物会点燃tnt和营火一类的方块",
+    "tetra.stats.punch": "冲击",
+    "tetra.stats.punch.tooltip": "增加 §e%s 倍的击退距离",
+    "tetra.stats.infinity": "无限",
+    "tetra.stats.infinity.tooltip": "射击时不再消耗普通的箭，依然会消耗特殊的箭",
+    "tetra.stats.quickStrike": "迅捷",
+    "tetra.stats.quickStrike.tooltip": "在攻击冷却时将最小伤害减少到 §e%s§r",
+    "tetra.stats.counterweight": "配重",
+    "tetra.stats.counterweight.tooltip": "当物品已使用了 §e%s§r 点完整度，能够增加 0.15 的攻速。每相差一点增益减少 0.05。",
+    "tetra.stats.softStrike": "轻击",
+    "tetra.stats.softStrike.tooltip": "使用这个工具合成的物品获得额外的耐久",
+    "tetra.stats.fierySelf": "燃烧",
+    "tetra.stats.fierySelf.tooltip_short": "使用时持有者有概率着火，在寒冷群系中概率较低",
+    "tetra.stats.fierySelf.tooltip": "使用时持有者有概率着火，在寒冷群系中概率较低\n\n§7基础概率：§e%s§7\n着火持续时间： §e%s",
+    "tetra.stats.enderReverb": "末影回响",
+    "tetra.stats.enderReverb.tooltip": "使用时激怒附近的末影人，并且附近的末影人传送时持有者有概率被一起传送\n\n§7基础概率：§e%s§7",
+    "tetra.stats.toolbelt.quickslot": "快捷栏",
+    "tetra.stats.toolbelt.quickslot.tooltip": "有 §e%s§r 个快速插槽",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
    "tetra.stats.toolbelt.quickslot.tooltip": "有 §e%s§r 个快捷栏槽位",
```